src/lwc/geTemplateBuilder/geTemplateBuilder.js

```diff
@@ -421,6 +428,23 @@ export default class geTemplateBuilder extends NavigationMixin(LightningElement)
                 }
             ];
         });
+
+        this.includePaymentStatusDisplayField();
+    }
+
+    includePaymentStatusDisplayField() {
+        const hasPaymentStatusDisplayValueField =
+            hasNestedProperty(this._dataImportObject, FIELDS, apiNameFor(PAYMENT_STATUS_DISPLAY_VALUE));
+
+        if (this.isElevateCustomer && hasPaymentStatusDisplayValueField) {
+            this.availableBatchTableColumnOptions = [
+                ...this.availableBatchTableColumnOptions,
```

Comment:
Can the two be concatenated or appended here instead of using the spread?

Reply:
Definitely, updated!